src/components/support/text-ellipsis.ts

```diff
@@ -0,0 +1,8 @@
+export const textEllipsis = (text, maxLength) => {
```

Comment:
We don't want to apply ellipsis through css, do we ? Because for wider boxes, more text would fit so we could trigger the ellipsis through css by enforcing the bounding box/height instead of truncating the actual text.

I bet you had already some thoughts on this, can you give me some context for the chose truncation approach?